build/go_build.sh

```diff
@@ -15,7 +15,7 @@ function build {
 	name=$3
 	echo "build os=${goos} arch=${goarch}" > /dev/stderr
 
-	CGO_ENABLED=0 GOOS=$goos GOARCH=$goarch go build -o $bin_dir/${name} -ldflags "-X main.BUILT_AT=$CURRENT_DATE -X=main.REVISION=$REVISION -X=main.PHRASEAPP_CLIENT_VERSION=$VERSION -X=main.LIBRARY_REVISION=$LIBRARY_REVISION -extldflags '-static'" .
+	CGO_ENABLED=0 GOOS=$goos GOARCH=$goarch go build -o $bin_dir/${name} -ldflags "-X=main.REVISION=$REVISION -X=main.PHRASEAPP_CLIENT_VERSION=$VERSION -X=main.LIBRARY_REVISION=$LIBRARY_REVISION -extldflags '-static'" .
```

Comment:
include the new information here too:
`... -X main.LAST_CHANGE=$CHANGE_DATE`

Reply:
I think it does not provide that much value as we still include the git commit hash in the build info

Reply:
It may not provide extra information, but it is more human readable (since its a timestamp)